server/routes/overview/overviewController.test.ts

```diff
@@ -367,6 +375,7 @@ describe('overviewController', () => {
         tab: expectedTab,
         functionalSkills: expectedFunctionalSkills,
         completedInPrisonEducation: expectedCompletedInPrisonEducation,
+        completedEducationRecordsWithinLast12Months: expectedCompletedEducationRecordsWithinLast12Months,
```

Comment:
The changes to the test here dont appear to be testing/asserting that the prison name lookup was performed? Or have I missed something? 🤔